tests/shared/pages/InboxPage.ts

```diff
@@ -42,14 +42,16 @@ export class ModalPage {
 
   async subscribe(nth: number) {
     await this.page.locator('.AppCard__body > .AppCard__body__subscribe').nth(nth).click()
+    await this.page.getByText('Subscribed to', { exact: false }).isVisible()
   }
 
-  async unsubscribe() {
-    await this.page.getByRole('button', { name: 'Subscribed' }).click()
-    await this.page.getByRole('button', { name: 'Subscribed' }).isHidden()
-    await this.page.getByRole('button').first().click()
-    await this.page.getByRole('button', { name: 'Unsubscribe' }).click()
+  async unsubscribe(nth: number) {
```

Comment:
I like the higher-level approach to selecting elements (like without using class names, etc). But at the same time, it doesn't really safe to me as well.

I.e: We are expecting the test runner to click to dropdown trigger on the dApp's page. 

Prev selection:
```js
await this.page.getByRole('button').first().click()
````

This seems a very fragile way to select it. UI changes through time can easily break it. Instead, we'd better to select it more specific way:

```js
await this.page.locator('.AppNotificationsHeader__wrapper > .Dropdown').click()
```